src/pages/ManhuaUS/main.ts

```diff
@@ -0,0 +1,89 @@
+import { pageInterface } from '../pageInterface';
+
+export const ManhuaUS: pageInterface = {
+  name: 'ManhuaUS',
+  domain: 'https://manhuaus.com',
+  languages: ['English'],
+  type: 'manga',
+  isSyncPage(url) {
+    return utils.urlPart(url, 5).length > 0;
+  },
+  sync: {
+    getTitle(url) {
+      return j.$(j.$('div.c-breadcrumb-wrapper ol.breadcrumb li a')[1]).text().trim();
+    },
+    getIdentifier(url) {
+      return utils.urlPart(url, 4);
+    },
+    getOverviewUrl(url) {
+      return url.split('/').slice(0, 5).join('/');
+    },
+    getEpisode(url) {
+      const episodePart = utils.urlPart(url, 5);
+      if (episodePart.length === 0) return NaN;
+
+      const temp = episodePart.match(/chapter-\d+/gim);
+      if (!temp || temp.length === 0) return NaN;
+
+      return Number(temp[0].replace(/\D+/g, ''));
+    },
+    nextEpUrl(url) {
+      return j
+        .$(
+          'div.entry-header.header > div > div.select-pagination > div.nav-links > div.nav-next > a.next_page',
```

Comment:
Would be nice to make this less specific. Because a small page change would break this selector